browser/ephemeral_storage/ephemeral_storage_service_factory.cc

```diff
@@ -46,21 +50,18 @@ void EphemeralStorageServiceFactory::RegisterProfilePrefs(
 
 KeyedService* EphemeralStorageServiceFactory::BuildServiceInstanceFor(
     content::BrowserContext* context) const {
-  if (!base::FeatureList::IsEnabled(
```

Comment:
why remove the check?

Reply:
> why remove the check?

because this is now used by the general Ephemeral Storage feature. It's possible to check all ES-related features here. Let me do that actually.